astro.config.ts

```diff
@@ -1,6 +1,6 @@
-import { defineConfig } from "astro/config"
-
+import netlify from "@astrojs/netlify/functions"
```

Comment:
Convert this to `astro.config.mjs` while we're at it

*Technically* `.ts` and other file extensions are supported but that's always been a weird headache, I know the idea was floated at least once to only support `.mjs` in the future (not currently planned though)